app/controllers/JobsController.scala

```diff
@@ -380,22 +379,20 @@ class JobsController @Inject()(
           _ <- Fox.runIf(userOrganization.pricingPlan == PricingPlan.Basic) {
             bool2Fox(animationJobOptions.movieResolution == MovieResolutionSetting.SD) ?~> "job.renderAnimation.resolutionMustBeSD"
           }
-          layerName = animationJobOptions.layerName
-          _ <- datasetService.assertValidLayerName(layerName)
-          _ <- Fox.runOptional(animationJobOptions.segmentationLayerName)(datasetService.assertValidLayerName)
-          exportFileName = s"webknossos_animation_${formatDateForFilename(new Date())}__${datasetName}__$layerName.mp4"
+          colorLayerName = animationJobOptions.colorLayerName
+          _ <- datasetService.assertValidLayerName(colorLayerName)
+          exportFileName = s"webknossos_animation_${formatDateForFilename(new Date())}__${datasetName}__$colorLayerName.mp4"
           command = JobCommand.render_animation
           commandArgs = Json.obj(
             "organization_name" -> organizationName,
             "dataset_name" -> datasetName,
             "export_file_name" -> exportFileName,
             "user_auth_token" -> userAuthToken.id,
-            "layer_name" -> animationJobOptions.layerName,
-            "segmentation_layer_name" -> animationJobOptions.segmentationLayerName,
+            "is_view_mode" -> animationJobOptions.isViewMode,
+            "color_layer_name" -> animationJobOptions.colorLayerName,
```

Comment:
note that if the command args change, the job list view code in the frontend also needs to be adapted (currently the existing layer_name is used to render the job description. compare also getJobs in admin_rest_api

Reply:
we also need to ensure backwards compatibility for this job list view rendering (or migrate old jobs in postgres)

Reply:
Good catch. I wasn't aware of this connection... will undo the renaming then.